teachopencadd/talktorials/019_One_Hot_Encoding/talktorial.ipynb

```diff
@@ -0,0 +1,2926 @@
+{
```

Comment:
<p>Should this packages be the top? I'm not sure.</p><p>I saw a rule that all <code>import ...</code> lines are at the top (practice part) cell, ordered by standard library / 3rd party packages / our own (<code>teachopencadd.*</code>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Done:)</p><p> </p>